scripts/release-notes/gen_ec_release_notes.sh

```diff
@@ -12,10 +12,15 @@ DEST_DIR="${DEST_DIR:-${DEFAULT_DEST_DIR}}"
 
 if [ ! -d "${DEST_DIR}" ]; then
     echo "Setting up virtualenv in ${DEST_DIR}"
-    python3 -m venv "${DEST_DIR}"
+    python3 -m venv --system-site-packages "${DEST_DIR}"
     "${DEST_DIR}/bin/python3" -m pip install --upgrade pip
     "${DEST_DIR}/bin/python3" -m pip install -r "${SCRIPT_DIR}/requirements.txt"
 fi
 source "${DEST_DIR}/bin/activate"
 
-python3 "${SCRIPT_DIR}/gen_ec_release_notes.py" "$@"
+if [ "$#" -ne 0 ] && [ "$1" == "--rhocp" ]; then
+    shift
+    python3 "${SCRIPT_DIR}/gen_release_notes.py" "$@"
+else
+    python3 "${SCRIPT_DIR}/gen_ec_release_notes.py" "$@"
```

Comment:
is `gen_ec_release_notes.py` only for EC? or for RC and EC versions?

Reply:
Both ECs and RCs. I renamed the scripts to make it more clear